mcs/class/corlib/Test/System/MathTest.cs

```diff
@@ -405,7 +405,7 @@ public void TestFloor ()
 		public void TestIEEERemainder ()
 		{
 			double a = Math.IEEERemainder (y, x);
-			double b = 0.0050000000000007816;
+			double b = 0.0050000000000010592;
 
```

Comment:
The previous value would fail the first Assert (it would also fail on .NET). The new value make this assert pass with the imported referencesource and on .NET.
@miguel @kumpera: as it's mainly a compatibility change, more than a correctness change, which one should we favor : compatibility with .NET, or correctness to the IEEE 754 standard?